service/history/api/replication/stream.go

```diff
@@ -290,3 +326,35 @@ Loop:
 		},
 	})
 }
+
+func (f *TaskConvertorImpl) Convert(
+	task tasks.Task,
+) (*replicationspb.ReplicationTask, error) {
+	if namespaceEntry, err := f.NamespaceCache.GetNamespaceByID(
+		namespace.ID(task.GetNamespaceID()),
+	); err == nil {
+		shouldProcessTask := false
+	FilterLoop:
+		for _, targetCluster := range namespaceEntry.ClusterNames() {
+			if f.SourceClusterShardID.ClusterName == targetCluster {
+				shouldProcessTask = true
+				break FilterLoop
```

Comment:
why break with label?

Reply:
to me break with label will make the logic easier to read